docs/cugraph/source/tutorials/cugraph_notebooks.md

```diff
@@ -1,6 +1,6 @@
 # cuGraph Notebooks
 
-![GraphAnalyticsFigure](img/GraphAnalyticsFigure.jpg)
+![GraphAnalyticsFigure](https://github.com/rapidsai/cugraph/tree/main/img/GraphAnalyticsFigure.jpg)
```

Comment:
Do we need to use the URL like this now, or can we continue to use the relative path like we were doing before this change?  Seems like using the relative path might be better for working on the docs locally and/or offline.